### ISSUE TITLE
Converting large files is not feasible due to high memory usage

### ISSUE THREAD

**Issue by dylanpdx** [AlfredoSequeida/fvid#30]
This tool works great for small files, but trying to convert a relatively large file (1GB) will cause a MemoryError, even with 32 GB of RAM

**Comment by Theelgirl** [comment]
Indeed. There's a bit of a tradeoff between memory and speed, and we've mostly gone the speed route since it was envisioned that people wouldn't try to encode a file above a few tens of megabytes. If there's an interest in adding modes for memory or speed, I'd be happy to work on that. For now though, can you run a memory profiler (scalene or memprof work well) to tell us where the code uses most of the memory on your machine, or send the 1GB file here so I can test?
